modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -71,12 +68,36 @@ The following Technology Preview features are enabled by this feature set:
 ** `PinnedImages`
 ** `PlatformOperators`
 ** `ServiceAccountTokenNodeBinding`
-** `ServiceAccountTokenNodeBindingValidation`
-** `ServiceAccountTokenPodNodeInfo`
 ** `TranslateStreamCloseWebsocketRequests`
 ** `UpgradeStatus`
 ** `VSphereMultiVCenters`
 ** `VolumeGroupSnapshot`
+** `AdditionalRoutingCapabilities`
+** `BootcNodeManagement`
```

Comment:
@sheriff-rh Is everything in the upstream list supposed to be replicated here? It's missing `AutomatedEtcdBackup` and `CSIDriverSharedResource` and a bunch 'o stuff after `DNSNameResolver`